orthoinference/src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -75,14 +75,15 @@ public static void inferEvents(Properties props, String species) throws Exceptio
 		setDbAdaptors(dbAdaptor);
 
 		releaseVersion = props.getProperty("releaseNumber");
-		String pathToOrthopairs = props.getProperty("pathToOrthopairs");
+		String pathToOrthopairs = props.getProperty("pathToOrthopairs") + releaseVersion + "/";
```

Comment:
Similar to the `Paths.get(...).toString()` comment above so the trailing slash is not hard-coded